components/release.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
```

Comment:
Maybe this script can remain optional part of the release. The code of these components rarely changes. Some people are getting a bit confused since we release new images and update the components and samples, while there are no real changes.